indico/modules/events/timetable/client/js/dnd/dnd.tsx

```diff
@@ -495,11 +497,11 @@ export function useDraggable({id, fixed = false}: {id: string; fixed?: boolean})
     };
   }, [id, fixed, registerDraggable, unregisterDraggable]);
 
-  const transform = (draggable || {}).transform;
-  const rect = (draggable || {}).rect;
-  const initialScroll = (draggable || {}).initialScroll;
-  const mouse = (draggable || {}).mouse;
-  const offset = (draggable || {}).initialOffset;
+  const transform = (draggableData || {}).transform;
+  const rect = (draggableData || {}).rect;
+  const initialScroll = (draggableData || {}).initialScroll;
+  const mouse = (draggableData || {}).mouse;
+  const offset = (draggableData || {}).initialOffset;
```

Comment:
Not very relevant for this PR but it might have made sense to do something like

```suggestion
  const {transform, rect, initialScroll, mouse, offset} = draggableData || {};
```